src/me/mrCookieSlime/Slimefun/Setup/SlimefunSetup.java

```diff
@@ -1756,6 +1794,56 @@ else if (b.getType().name().endsWith("_SHULKER_BOX")) {
 			}
 		});
 
+		new SlimefunItem(Categories.TOOLS, SlimefunItems.EXPLOSIVE_SHOVEL, "EXPLOSIVE_SHOVEL", RecipeType.MAGIC_WORKBENCH,
+		new ItemStack[] {null, SlimefunItems.SYNTHETIC_DIAMOND, null, null, new ItemStack(Material.TNT), null, null, SlimefunItems.FERROSILICON, null},
+		new String[] {"damage-on-use"}, new Object[] {Boolean.FALSE })
+		.register(true, new BlockBreakHandler() {
+
+			@Override
+			public boolean onBlockBreak(BlockBreakEvent e, ItemStack item, int fortune, List<ItemStack> drops) {
+				if (SlimefunManager.isItemSimiliar(item, SlimefunItems.EXPLOSIVE_SHOVEL, true)) {
+					e.getBlock().getWorld().createExplosion(e.getBlock().getLocation(), 0.0F);
+					e.getBlock().getWorld().playSound(e.getBlock().getLocation(), Sound.ENTITY_GENERIC_EXPLODE, 1F, 1F);
+					for (int x = -1; x <= 1; x++) {
+						for (int y = -1; y <= 1; y++) {
+							for (int z = -1; z <= 1; z++) {
+								Block b = e.getBlock().getRelative(x, y, z);
+								boolean correctType = false;
+								for (Material mat : MaterialCollections.getAllFishItems()) {

```

Comment:
getAllFishItems ????

Reply:
That was a placeholder method so I could build the plugin to test if setup and etc. works.

Reply:
Well, use CS-CoreLib2's method then.

All you have to do is update the pom.xml and change the version of CS-CoreLib to 0.3.8
And add the path of the materials package in the shade-plugin section